### PR TITLE
add pack stamper beta

### PR DESCRIPTION
**Pull request by Cryptocojak**
thanks to @rahdoht for doing the heavy lifting on this, and @karpapy for being based.

### CODE REVIEW

**Comment by karpapy** [comment]
image

File: src/components/Crop.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    };
+    reader.readAsDataURL(files[0]);
+  };
+
+  return (
+    <div style={{ display: "flex", flexWrap: "wrap" }}>
+      <div style={{ flexBasis: "50%", padding: "10px" }}>
+        <div style={{ marginBottom: "10px" }}>
+          <input type="file" onChange={onChange} />
+        </div>
```
Comment (karpapy) [inline src/components/Crop.jsx:82]: For example, we can make these `<div>`s in the chakra style like 
```jsx
<Box display="flex" flexWrap="wrap">
```
and chakra also has an input element like 
```jsx
<Input type="file" onChange={onChange} />
```

File: src/components/Render.jsx [hunk 2]
```diff
@@ -117,7 +117,7 @@ export const RenderComponent = ({ setUserDidRender }) => {
   const [showCustomize, setShowCustomize] = useState(false)
   const [filteredData, setFilteredData] = useState(null)
   const [selectedAttributes, setSelectedAttributes] = useState({})
-  const [textTitle, setTextTitle] = useState('cigbot1111')
+  const [textTitle, setTextTitle] = useState('cigwave summer')
```
Comment (karpapy) [inline src/components/Render.jsx:120]: hot cig summer

**Review by karpapy — COMMENTED** [review]
Some real great engineering @Cryptocojak ! can we go full chakra so that our inner light stays shinning ❤️

File: src/components/Crop.jsx (hunk 1 above)
Comment (Cryptocojak) [reply src/components/Crop.jsx:82]: based and chakra pilled (thanks let’s just merge this, and I’ll go ahead and open the comments as an issue and we’ll address them in a separate PR)

**Comment by karpapy** [comment]
MERGED!!!

**Pull request merged**